src/chapters/jdbc-auth.xml

```diff
@@ -227,20 +227,13 @@ Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
                     is named <filename>upgrade-pre-<replaceable>VERSION</replaceable>.sql</filename>
                     where <replaceable>VERSION</replaceable> is the version of Guacamole where those
                     changes were introduced. They need to be run when you are upgrading from a
-                    version of Guacamole older than <replaceable>VERSION</replaceable>:</para>
-                <informalexample>
-                    <screen><prompt>$</prompt> <userinput>ls schema/upgrade/</userinput>
-<computeroutput>upgrade-pre-0.8.2.sql   upgrade-pre-0.9.13.sql  upgrade-pre-0.9.8.sql
-upgrade-pre-0.9.10.sql  upgrade-pre-0.9.6.sql   upgrade-pre-0.9.9.sql
-upgrade-pre-0.9.11.sql  upgrade-pre-0.9.7.sql</computeroutput>
-<prompt>$</prompt></screen>
-                </informalexample>
+                    version of Guacamole older than <replaceable>VERSION</replaceable>.</para>
                 <para>These scripts are incremental and, when relevant, <emphasis>must be run in
                         order</emphasis>. For example, if you are upgrading an existing database
-                    from version 0.9.10-incubating, you would need to run the
-                        <filename>upgrade-pre-0.9.11.sql</filename> script (because 0.9.10 is older
-                    than 0.9.11), followed by the <filename>upgrade-pre-0.9.13.sql</filename> script
-                    (because 0.9.10 is also older than 0.9.13):</para>
+                    from version 0.9.10-incubating to version 0.9.13-incubating, you would need to
```

Comment:
Was there a reason to leave the -incubating reference in here and not make the example for 0.9.14, without that?  I think it's good either way, just want to make sure.

Reply:
My only reason was to avoid having to rewrite the example.

Reply:
That's fair.  Like I said, fine with me, just wanted to make sure.

Reply:
(...with the rest of the rewording here to avoid having to rewrite the example for future releases, as well)